packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -56,10 +66,43 @@ test('start', async t => {
   const seat = await E(zoe).offer(inv);
   t.is(
     await E(seat).getOfferResult(),
-    'noop; evidence was pushed in the invitation maker call',
+    'noop; nothing should be expected from this offer',
   );
 });
 
+test('oracles', async t => {
```

Comment:
I suggest a test assertion that says a bit more about what we're testing to orient readers:

```suggestion
test('oracles have closely-held rights to submit evidence of CCTP transactions', async t => {
```

Or "access privileges for the transaction feed oracles" as it says in the PR description, though I prefer assertions to noun phrases.